ibc/host_introspection_test.go

```diff
@@ -0,0 +1,271 @@
+package ibc
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	ics23 "github.com/cosmos/ics23/go"
+	light_client_types "github.com/pokt-network/pocket/ibc/client/light_clients/types"
+	client_types "github.com/pokt-network/pocket/ibc/client/types"
+	"github.com/pokt-network/pocket/ibc/types"
+	"github.com/pokt-network/pocket/shared/codec"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+func TestHost_GetCurrentHeight(t *testing.T) {
+	_, _, _, _, ibcMod := prepareEnvironment(t, 1, 0, 0, 0)
+	cm := ibcMod.GetBus().GetClientManager()
+
+	// get the current height
+	height, err := cm.GetCurrentHeight()
+	require.NoError(t, err)
+	require.Equal(t, uint64(1), height.GetRevisionNumber())
+	require.Equal(t, uint64(0), height.GetRevisionHeight())
+
+	// increment the height
```

Comment:
REQUEST: Can you add a few more comments like this throughout the test to make life easier for me (the reader)?